build/ci/github_actions/driver/deploy_driver.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -xel
+set +o pipefail
+
+expected_node_image=$node_repository_for_test:$driver_images_tag
+expected_controller_image=$controller_repository_for_test:$driver_images_tag
+
+install_worker_prerequisites() {
+  kind_node_name=`docker ps --format "{{.Names}}"`
+  docker exec -i $kind_node_name apt-get update
+  docker exec -i $kind_node_name apt -y install open-iscsi
```

Comment:
is this for automatic iscsi login?
what happens if we don't install it?

Reply:
the node is failing if the host doesn't have FC or iscsi, so it is easier to install iscsi